cdn.py

```diff
@@ -8,8 +8,11 @@
 @roles('class-cache')
 def fastly_purge(*args):
     "Purge items from Fastly, eg \"/one,/two,/three\". Wildcards not supported."
-    govuk_fastly = 'http://www-gov-uk.map.fastly.net'
-    hostnames_to_purge = ['www.gov.uk', 'assets.digital.cabinet-office.gov.uk']
+    if env.environment == 'production':
+        hostnames_to_purge = ['www.gov.uk', 'assets.digital.cabinet-office.gov.uk']
+    elif env.environment == 'staging':
+        hostnames_to_purge = ['www.staging.alphagov.co.uk', 'assets.staging.alphagov.co.uk']
```

Comment:
Purging from the Staging Fastly service would be ineffective until we redeploy the Fastly service to use the new domain, but I don't think that's a problem as we rarely use that service.

If we keep this as-is we might want to add a `FIXME` as a reminder to change to the new domain after the migration.